### PR TITLE
you can't run 'yarn install' in root. You have to run it in \app with…

### PR DESCRIPTION
**Pull request by JustinGuerra**
…in root.

### CODE REVIEW

**Comment by JustinGuerra** [comment]
Screen Shot 2019-03-12 at 5 37 09 PM

**Comment by JustinGuerra** [comment]
This can be corrected by running the install command as follows "sudo yarn install", but now there is an issue with file names. 

image

**Comment by DanielHix** [comment]
rename the App.tsx file to app.tsx file, and also do the same with the css and test.tsx files if you don't mind sir. Don't forget to fix the app.css reference in the app.tsx file. I have this done on my current branch, but only noticed it when I started on a new feature.

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ You will need to install the following software onto your machine.
 
 - [Yarn](https://yarnpkg.com/en/)
 
-Once those are installed you will then run ```yarn install``` in the root of the cloned repository. Once this is done you can type ```yarn start``` and run the project. The reload and developer tools can be found under the "view" tab for now (or by their short cuts).
+Once those are installed you will then run ```yarn install``` in the \app directory of the cloned repository. Once this is done you can type ```yarn start``` and run the project. The reload and developer tools can be found under the "view" tab for now (or by their short cuts).
```
Comment (DanielHix) [inline README.md]: you also only need to run ```yarn``` if you'll also make that change (since you're fixing the readme in this PR anyway).

**Pull request merged**